### PR TITLE
Add RadioField

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Demo with a required field and without.

form

### CODE REVIEW

File: src/components/RadioField.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+export default {
+    name: 'RadioField',
+    mixins: [ValidateField],
+    data() {
+        let options = [
+            { text: 'Vrai', value: 'true' },
+            { text: 'Faux', value: 'false' },
+        ]
+
+        if (this.containsKey(this.field, "constraints") && this.containsKey(this.field.constraints, "required") && this.field.constraints.required) {
```
Comment (AntoineAugusti) [inline src/components/RadioField.vue]: Is there a shorthand for this?

File: src/views/SchemaForm.vue [hunk 2]
```diff
@@ -173,6 +175,13 @@ export default {
             })
             instance.$mount()
             return this.$refs.container.appendChild(instance.$el)
+          } else if (isBoolean) {
```
Comment (AntoineAugusti) [inline src/views/SchemaForm.vue]: If you know how to do this with fewer lines

Comment (abulte) [reply src/views/SchemaForm.vue]: I don't know if it answers your question, but I think the following code could be factorized in a function that takes a Field as parameter.

```
             const RadioFieldClass = Vue.extend(RadioField)
             let instance = new RadioFieldClass({
                 propsData: {field: field}
             })
             instance.$mount()
             return this.$refs.container.appendChild(instance.$el)
```

File: src/components/RadioField.vue (hunk 1 above)
Comment (abulte) [reply src/components/RadioField.vue]: `if (this.field.constraints && this.field.constraints.required)` doesn't work?

If not maybe you can extract the constraints in a list in the parent Class and just do a `indexOf`.

**Pull request merged**